examples/pr_inversion_example.py

```
@@ -0,0 +1,85 @@
+4. Run auto_invert() and read the diagnostics
+5. Control n_terms/alpha explicitly and export the result
+"""
+
+import numpy as np
+
+from sans_fitter import data_ops, pr_inversion
+
+# ============================================================================
+# Part 1: simulate a protein-like sphere dataset (R = 60 Å, 2% noise)
+# ============================================================================
+# The sphere form factor is analytic: I(q) = I(0)·[3(sin(u) − u·cos(u))/u³]²
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `−` (MINUS SIGN). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)